File: task4.py
def get_number(index):
  if(index%2==0):
    print(int(index / 2)*3)
  elif(not index%2==0):
    print(4+int(index / 2)*3)
  
get_number(1) # print 4 
get_number(5) # print 10 
get_number(10)# print 15

File: task4.js
function getNumber(index){
  if(index%2==0){
    console.log(Math.floor(index / 2)*3)
  }else if(!index%2==0){
    console.log(4+Math.floor(index / 2)*3)
  }
}

getNumber(1); // print 4 
getNumber(5); // print 10 
getNumber(10); // print 15
